src/utils.rs

```diff
@@ -214,6 +215,23 @@ pub fn parse_tag(input: Option<&str>) -> Option<String> {
     }
 }
 
+pub fn get_uuid_from_url(url: &str) -> String {
+    if let Ok(url) = Url::parse(url) {
+        let path_segments: Vec<_> = url.path_segments().unwrap().collect();
+        if let Some(uuid_segment) = path_segments.get(3) {
+            if uuid_segment.len() == 36 {
```

Comment:
Might be a little cleaner to define a constant here like UUID_LENGTH OR UUID_SEGMENT_LENGTH instead of the raw `36`.

Reply:
Thanks for taking a look!

I've had a chance to think about this a bit too, this is too specific to be in the utils file. If I tried using it for something other than projects it would fail or be inconsistent since we string ids and the ids, for example parameters, could be at another position in the vector.

I think having the parent project's id would be useful, but it may not be worth it unless you or someone else has some advice.

Reply:
Also, you are correct, that's a better way re: constant